src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/output/StandardMovesSheet.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.output
+
+import org.apache.poi.ss.usermodel.Sheet
+import org.apache.poi.ss.usermodel.Workbook
+import uk.gov.justice.digital.hmpps.pecs.jpc.calculator.MovePrice
+import uk.gov.justice.digital.hmpps.pecs.jpc.pricing.Supplier
+import java.time.LocalDate
+import java.util.concurrent.atomic.AtomicInteger
+
+class StandardMovesSheet(workbook: Workbook, private val date: LocalDate, supplier: Supplier) {
+
+    private val sheet: Sheet = workbook.getSheet("Standard")!!
+    private val index: AtomicInteger = AtomicInteger(10)
+
+    init {
+       addHeading(date, supplier)
+    }
+
+    private fun addHeading(date: LocalDate, supplier: Supplier) {
+        sheet.getRow(0).createCell(1).setCellValue(date)
```

Comment:
@jcavell so whilst this populates the cell with the date it looks like further work is need to ensure the correct styling is appled.